src/App.js

```diff
@@ -15,17 +15,10 @@ function App() {
           className={ !isShown ? "App-logo" : "App-logo-reversed" }
           alt="logo"
         />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
+        <p style={{textAlign: "center"}}>
+          <h1>Welcome to the hell marine!</h1>
         </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
+        <button class="button button3">I'm too young to die!</button>
```

Comment:
Why do we use 2 classes here?

Reply:
Because if use 1 class the button behaviour will change.

Reply:
nvm, solved that for you